deps/npm/node_modules/lru-cache/foo.js

```diff
@@ -0,0 +1 @@
+console.log(require("."))
```

Comment:
\code review\ but these files to not appear in https://github.com/isaacs/node-lru-cache - will open a PR on npm.